From fedcad308db06f5a38ee191a806131a927d94c1d Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 14:58:19 -0400
Subject: [infiniband] usnic: Append documentation to usnic_transport.h and cleanup

Message-id: <50986578b5b5deeef3f96d286be8771587d787d8.1411569661.git.dledford@redhat.com>
Patchwork-id: 96589
O-Subject: [Patch RHEL7 17/29] IB/usnic: Append documentation to usnic_transport.h and cleanup
Bugzilla: 916382
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Upinder Malhi <umalhi@cisco.com>

Add comment describing usnic_transport_rsrv port and remove
extraneous space from usnic_transport.c.

Signed-off-by: Upinder Malhi <umalhi@cisco.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 3108bccb3d9afbd32931d775f5dd5ee157eaa5a9)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/usnic/usnic_transport.c b/drivers/infiniband/hw/usnic/usnic_transport.c
index 73dffc9..9182cc8 100644
--- a/drivers/infiniband/hw/usnic/usnic_transport.c
+++ b/drivers/infiniband/hw/usnic/usnic_transport.c
@@ -184,7 +184,6 @@ int usnic_transport_sock_get_addr(struct socket *sock, int *proto,
  return 0;
 }
 
-
 int usnic_transport_init(void)
 {
  roce_bitmap = kzalloc(ROCE_BITMAP_SZ, GFP_KERNEL);
diff --git a/drivers/infiniband/hw/usnic/usnic_transport.h b/drivers/infiniband/hw/usnic/usnic_transport.h
index 21371bb..7e5dc6d 100644
--- a/drivers/infiniband/hw/usnic/usnic_transport.h
+++ b/drivers/infiniband/hw/usnic/usnic_transport.h
@@ -28,6 +28,10 @@ const char *usnic_transport_to_str(enum usnic_transport_type trans_type);
  */
 int usnic_transport_sock_to_str(char *buf, int buf_sz,
      struct socket *sock);
+/*
+ * Reserve a port. If "port_num" is set, then the function will try
+ * to reserve that particular port.
+ */
 u16 usnic_transport_rsrv_port(enum usnic_transport_type type, u16 port_num);
 void usnic_transport_unrsrv_port(enum usnic_transport_type type, u16 port_num);
 /*
-- 
1.7.1